examples/charts/area.py

```diff
@@ -1,6 +1,7 @@
 from collections import OrderedDict
-from bokeh.charts import Area
-from bokeh.plotting import show, output_file
+
+from bokeh.charts import Area, show, output_file
```

Comment:
I like the centralize thing... but as I said in other PR, I also see value in this... mainly from the pragmatic perspective...

>I would say that maybe we do move the functions to their own "io" api module, and make a point to always import the entire "io" api module into the other interfaces.

Yep...